packages/test/src/integration-tests.ts

```diff
@@ -391,6 +392,21 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     t.pass();
   });
 
+  test('query not found', async (t) => {
+    const { client } = t.context;
+    const workflow = await client.start(workflows.unblockOrCancel, {
+      taskQueue: 'test',
+      workflowId: uuid4(),
+    });
+    await workflow.signal(workflows.unblockSignal);
+    await workflow.result();
+    await t.throwsAsync(workflow.query('not found'), {
+      instanceOf: QueryNotRegisteredError,
+      message:
+        '3 INVALID_ARGUMENT: Workflow did not register a handler for not found. Registered queries: [__stack_trace isBlocked]',
```

Comment:
Is this part helpful? `3 INVALID_ARGUMENT: ` Wonder if we should remove

Reply:
I didn't want to mess with the returned message, I guess it's okay to trim.